packages/blade/src/components/Box/BaseBox/types/props.types.ts

```diff
@@ -129,22 +131,19 @@ type BoxProps = Partial<
     LayoutProps &
     FlexboxProps &
     PositionProps &
-    GridProps & { children?: React.ReactNode | React.ReactNode[] }
+    GridProps &
```

Comment:
Seems like this way we would also show grid props auto suggestions for native modules. Can we leverage TS module suffixes to fine tune that?

Reply:
Both for us internally and consumer 🤔

Reply:
Done 🎉 bd61a389316043bce1e09610c90ef16734867795

btw how do we tell vscode to pick native autocomplete instead of web autocomplete? locally it was always showing me web suggestions. This only worked on CLI when I ran `typecheck`

Reply:
Summoning @anuraghazra :godmode: ;)

Reply:
> btw how do we tell vscode to pick native autocomplete instead of web autocomplete? locally it was always showing me web suggestions

You need to change the tsconfig [here](https://github.com/razorpay/blade/blob/cd78e9b3b79777073f96e433eccf908f9ea73fcc/packages/blade/tsconfig.json#L12-L14)

add .native and it will work, tho you can't add both.